src/confluent_kafka/schema_registry/protobuf.py

```diff
@@ -158,6 +158,10 @@ class ProtobufSerializer(object):
     |                                     |          | Schema Registry subject names for Schema References  |
     |                                     |          | Defaults to reference_subject_name_strategy          |
     +-------------------------------------+----------+------------------------------------------------------+
+    |                                     |          | Specifies whether the Protobuf serializer should     |
+    | ``use.deprecated.format``           | bool     | serialize message indexes without zig-zag encoding.  |
+    |                                     |          | Defaults to False.                                   |
```

Comment:
It would be good to explain when this is needed, and also outline compatibility issues with this property. E.g., messages produced with version <= 1.7.0 will not be compatible with bub-bub-bub.